torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -1034,6 +1035,20 @@ Tensor TensorExprKernel::bindInput(const torch::jit::Value* input) {
     }
   };
 
+  auto setQinfoAsNeeded = [&](c10::TensorTypePtr tt, BufHandle inBuffer) {
+    auto scalarType = tt->scalarType();
+    if (scalarType == at::ScalarType::QInt8 || scalarType == at::ScalarType::QUInt8) {
+      VarHandle scale(input_name_map_[input] + "_scale", kDouble);
+      bufferArgs_.emplace_back(scale);
+      VarHandle zero_point(input_name_map_[input] + "_zero", kLong);
+      bufferArgs_.emplace_back(zero_point);
+      inBuffer.node()->set_qscale(scale.node());
+      inBuffer.node()->set_qzero(zero_point.node());
+      qtensorInputIndex_.emplace(input_name_map_[input] + "_scale", runArgs_idx);
+      runArgs_idx += 2;
```

Comment:
Is it possible to use the size of `bufferArgs_` to calculate the `runArgs_idx` instead of explicitly tracking it with an extra variable?